homeworks/serhii.ripchanskyi_Ripchanskiy/13-ng-forms/src/app/components/login-form/login-form.component.scss

```diff
@@ -0,0 +1,24 @@
+.error {
+  color: #B50404;
+  text-align: end;
+  font-size: 12px;
+}
+
+input {
```

Comment:
That's a bad practice to query by tag name. So please, create a dedicated class for inputs :slightly_smiling_face: